packages/shorebird_cli/lib/src/code_push_client_wrapper.dart

```diff
@@ -220,13 +257,19 @@ Please create a release using "shorebird release" and try again.
       'Fetching $arch artifact',
     );
     try {
-      final artifact = await codePushClient.getReleaseArtifact(
+      final artifacts = await codePushClient.getReleaseArtifacts(
```

Comment:
can `getReleaseArtifact` call `maybeGetReleaseArtifact` to avoid code duplication?

Reply:
That would simplify this a lot, but doing so would show a new `progress` for every artifact, which I don't think is what we want.

Reply:
I see what you're saying – I'm just punting that refactor to a future PR to keep this one reasonably sized